<?xml version="1.0"?>
<launch>

    <!-- parameter -->
    <arg name="shooting_arm"          default="shooting_arm" />
    <param name="/shoot_arm"           value="$(arg shooting_arm)" />

    <!-- connect left vx300s -->
    <include file="$(find arm_control)/launch/arm_control_real.launch">
            <arg name="robot_model"     value="vx300s" />
            <arg name="port"            value="/dev/ttyDXL" />
            <arg name="robot_name"      value="$(arg shooting_arm)" />
            <arg name="use_rviz"        value="false" />
    </include>

</launch>